crates/enarx-config/Enarx_toml.md

```diff
@@ -108,29 +108,22 @@ VAR1 = "var1"
 VAR2 = "var2"
 
 # Pre-opened file descriptors
-[[files]]
+[stdin]
 kind = "null"
 
-[[files]]
+[stdout]
 kind = "stdout"
 
-[[files]]
+[stderr]
 kind = "stderr"
 
 # A listen socket
-[[files]]
-name = "listen"
-kind = "listen"
+[listen.12345]
 prot = "tls" # or prot = "tcp"
-port = 12345
 
 # An outgoing connected socket
-[[files]]
-name = "connect"
-kind = "connect"
+[connect."127.0.0.1:23456"]
 prot = "tcp" # or prot = "tls"
-host = "127.0.0.1"
-port = 23456
```

Comment:
I guess, this will invalidate all drawbridge Enarx.tomls

Reply:
Yes

Reply:
That's why there's "adapt codex examples" follow-up.
We have to rewrite all codex examples to use runtime sockets